helper/stats/cpu.go

```diff
@@ -1,14 +1,21 @@
 package stats
 
 import (
+	"context"
 	"fmt"
 	"math"
 	"sync"
+	"time"
 
 	multierror "github.com/hashicorp/go-multierror"
 	"github.com/shirou/gopsutil/cpu"
 )
 
+const (
+	// We use this constant to override the built in 3 second timeout in gopsutil
```

Comment:
cpuInfoTimeout is the timeout used when gathering CPU info. This is used to override the default timeout in gopsutil which has a tendency to timeout on Windows.